UnraisinYourself/ViewController.swift

```diff
@@ -8,14 +8,42 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    let drinkButton = UIButton()
```

Comment:
This could probably be private unless we have another view or view controller that needs to access this same drinkButton :)